### PR TITLE
[NTR-483] Add SES configuration

### PR DESCRIPTION
**Pull request by miqs1992**
## Motivation and Context

https://eaflood.atlassian.net/browse/NTR-482
https://eaflood.atlassian.net/browse/NTR-483

## Description

I added [aws-sdk-rails gem](https://github.com/aws/aws-sdk-rails) to handle sending emails and a basic mailer.

## How Has This Been Tested?

1. Add `SES_ACCESS_KEY_ID` and `SES_SECRET_ACCESS_KEY` env variables
2. Go to AWS Console -> SES -> Email Addresses -> Verify a new email address
3. Add your email and click the link that will be sent to you
4. Set `SES_FROM_EMAIL` with some verified email
5. Run this in `rails console`
```
user = User.new
user.email = <your verified email>
UploadMailer.success_upload(user).deliver
```
6. You should receive en email

## Screenshots (if appropriate):

Screen Shot 2019-09-18 at 11 09 48

## Checklist:

- [x] It contains only changes required by issue (does not contain other PR)
- [x] Includes a link to an issue (if apply)
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: app/mailers/upload_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UploadMailer < ApplicationMailer
```
Comment (likeuwill) [inline app/mailers/upload_mailer.rb]: what about rdoc?

File: app/views/layouts/mailer.html.haml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=utf-8", "http-equiv" => "Content-Type"}/
```
Comment (likeuwill) [inline app/views/layouts/mailer.html.haml]: use new syntax

File: spec/mailers/previews/upload_mailer_preview.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Preview all emails at http://localhost:3000/rails/mailers/upload_mailer
+class UploadMailerPreview < ActionMailer::Preview
```
Comment (likeuwill) [inline spec/mailers/previews/upload_mailer_preview.rb]: rdoc

**Comment by miqs1992** [comment]
RDOC added

**Comment by miqs1992** [comment]
RDOC added

**Pull request merged**